docs/app/pages/home/landing-home.component.ts

```diff
@@ -6,15 +6,18 @@
 
 import { Component } from '@angular/core';
 import { NbThemeService } from '@nebular/theme';
+import {MetadataService} from '../../../../src/app/@core/utils/metadata.service';
 
 @Component({
   selector: 'ngx-landing-home',
   templateUrl: './landing-home.component.html',
   styleUrls: ['./landing-home.component.scss'],
 })
 export class LandingHomeComponent {
-
-  constructor(private themeService: NbThemeService) {
+  constructor(private themeService: NbThemeService, private metadataService: MetadataService) {
+    this.metadataService.updateDescription('Save more than $33,000 using ngx-admin for personal and commercial' +
+      ' use. The dashboard is based on Angular 9+ and Bootstrap 4+. Completely FREE and MIT licensed.');
+    this.metadataService.updateTitle('The most popular admin dashboard based on Angular 9+ and Nebular.');
```

Comment:
Title should be
`Ngx-admin - most popular admin dashboard on Angular 9+ and Nebular.`

Reply:
fix